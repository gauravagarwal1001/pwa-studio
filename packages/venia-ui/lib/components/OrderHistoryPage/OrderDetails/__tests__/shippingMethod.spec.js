import React from 'react';
import { createTestInstance } from '@magento/peregrine';

import ShippingMethod from '../shippingMethod';

const defaultProps = {
    data: {
        shipments: [
            {
                id: '1',
                tracking: [
                    {
                        carrier: 'Fedex',
                        number: 'FEDEX5885541235452125'
                    }
                ]
            }
        ],
        shipping_method: 'Free'
    }
};

test('should render properly', () => {
    const tree = createTestInstance(<ShippingMethod {...defaultProps} />);

    expect(tree.toJSON()).toMatchSnapshot();
});
