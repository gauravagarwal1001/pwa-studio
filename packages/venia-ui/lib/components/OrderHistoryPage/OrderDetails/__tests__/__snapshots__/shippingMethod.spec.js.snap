// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<div>
  <div>
    Shipping Method
  </div>
  <div />
  <div>
    Fedex Tracking: FEDEX5885541235452125
  </div>
</div>
`;
